CMakeFuncs.cmake

```diff
@@ -0,0 +1,627 @@
+# This is a set of common functions and settings for rippled
```

Comment:
if you prefer to hide this file a bit, I suppose we could move it to "Builds/CMake/" or something similar and that add that to the cmake include search path before importing it.